docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -47,27 +47,33 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
 
     - **Role**: Enter your Snowflake role.
 
-    - **Authenticator**: Enter the Snowflake database authenticator that you want to use to verify your Snowflake connection. 
+6. Optional. Make the following changes:
 
-5. Optional. Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
+    - Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
 
-6. Optional. Clear **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
+    - Select **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
 
 7. Click **Continue**.
 
 8. Select **Table Asset** or **Query Asset** and complete the following fields:
 
-    - **Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
-
     - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+    
+    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
 
     - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
 
-9. Optional. Select **Add another Data Asset** to add additional tables or queries and repeat step 7.
+9. Select the **Complete Asset** tab to create Expectations and run validations on the entire Data Asset, or select the **Batches** tab to create Expectations and run validations on subsets of Data Asset records and then complete the following fields:
```

Comment:
I think this should be reworded.  They aren't creating Expectations or running validations from either of these tabs, they're determining how the records in the Data Asset will be organized.
```suggestion
9. Select the **Complete Asset** tab to indicate that all records in the Data Asset should be provided to Expectations when validations are run, or select the **Batches** tab to specify subsets of Data Asset records that can be provided to Expectations when validations are run. If the **Batches** tab has been selected, complete the following fields:
```